wp-seo.php

```diff
@@ -1,5 +1,10 @@
 <?php
-/*
+/**
+ * @package    WPSEO
+ * @subpackage Main
+ */
+
+/**
```

Comment:
@Rarst should this be a docblock `/**`? Shouldn't this be a multiline comment `/*`?